lib/models/model.dart

```diff
@@ -0,0 +1,6 @@
+ // Default model
+abstract class Model {
+  Map<String, dynamic> toMap();
+  List<dynamic> toList(maps);
+  
+}
```

Comment:
New line at end of file please

Reply:
Have a look though all the files and do this

Reply:
I'll do it because I know it's what you're supposed to do but out of curiosity why?

Reply:
git hub is a bitch and fucks up some times... the newline is the reason. had to deal with it at verafin all the time.

Reply:
This can get funky for parsing with various tools. Each line has an end of line so that parsers understand. But if the last line does not have the end of line, a parser could end up hanging at the end, or throwing an error rather then its proper output